docs/dictionary/command/send.lcdoc

```diff
@@ -35,24 +35,24 @@ end timerIncrement
-object: Any object reference. If you don't specify an object, the message is set to the object whose handler contains the send command. If you specify a time, you must also specify an object.
+object: Any object reference. If you don't specify an object, the message is set to the object whose <handler> contains the send command. If you specify a time, you must also specify an object.
 time (integer): If you don't specify a unit of time, the default of ticks is used.
 
-The result: If you specify a <time>, the <message> is sent to the <object> after the specified time has elapsed. The <message> is added to the <pendingMessages> <function>. The ID of the <message> is <return|returned> by the <result> <function>. To cancel the <message>, use the <cancel> <command> with this ID.
+The result: If you specify a <time>, <the result> is the ID of the <message> in the <pendingMessages> <function> queue.
 
 Description:
-Use the <send> <command> to override the normal <message path>, or to delay a <command> until a specified time. 
+Use the <send> <command> to override the normal <message path>, or to delay a <command> until after a specified time.
 
-The <message> can be either the name of a <handler> or a <LiveCode> <statement>. If the <message> is a <literal string> containing more than one <word> (for example, if it includes <parameter|parameters>, or if it is a multi-word command), it must be enclosed in <double quote|double quotes>. 
+The <messageName> can be either the name of a <handler> or a <LiveCode> <statement>. If the <messageName> is a <literal string> containing more than one <word> (for example, if it includes <parameter|parameters>, or if it is a multi-word command), it must be enclosed in <double quote|double quotes>.
 
-Any parameters are evaluated before they are passed to the <send> <command>. For example, the following statement sends the <mouseUp> <message> with 2 as the first parameter:
+Any parameters are evaluated before they are passed to the <send> <command>. For example, the following statement sends the <mouseUp> <messageName> with 2 as the first parameter:
 
   send "mouseUp 1+1" to button "Example"
 
-When the send command is used the stack containing the target handler temporarily becomes the defaultStack. All object references in the <message> are <evaluate|evaluated> in the current context i. e. the defaultStack. Therefore references within the message that refer to "this card" or "this stack" will be referring to the card or stack where the target handler is located. 
+>*Important:* When the send command is used, the stack containing the target <handler> temporarily becomes the defaultStack. All object references in the <messageName> are <evaluate|evaluated> in the current context i. e. the defaultStack. Therefore references within the message that refer to "this card" or "this stack" will be referring to the card or stack where the target <handler> is located.
```

Comment:
I think `the <messageName>` is not quite right here as it's just a string - it should be `the target <handler>`.